src/components/v2/Toggle/index.tsx

```diff
@@ -10,14 +10,14 @@ export interface IToggleProps {
   value: boolean;
   className?: string;
 }
-
-const label = { inputProps: { 'aria-label': 'Switch' } };
+export const switchAriaLabel = 'Switch';
+const label = { inputProps: { 'aria-label': switchAriaLabel } };
 
 export const Toggle = ({ onChange, value, className }: IToggleProps) => {
   const styles = useStyles();
   return (
     <Switch
-      className={className}
+      className={`${className} toggle`}
```

Comment:
Just curious, what is the `toggle` class used for here?

Reply:
I don't remember why I added it but I didn't end up using it so it can be removed